src/main/java/uk/gov/justice/probation/courtcasematcher/model/mapper/CaseMapper.java

```diff
@@ -147,29 +153,29 @@ public CourtCase merge(Case incomingCase, CourtCase existingCourtCase) {
 
     public CourtCase newFromCourtCaseWithMatches(CourtCase incomingCase, MatchDetails matchDetails) {
 
-        ProbationStatusDetail statusDetail = matchDetails.getProbationStatusDetail();
```

Comment:
probation status will no longer be in `MatchDetails`. If there is one match, it will be in the offender match record as returned by offender-search. If there are more than 1 matches, then it will be set as a string here and not in `MatcherService` as it was before